scte35/segmentationdescriptor.go

```diff
@@ -247,7 +247,7 @@ func (d *segmentationDescriptor) CanClose(out SegmentationDescriptor) bool {
 // Equality in this sense means that two "in" events are duplicates
 // A lot of debate went in to what actually constitutes a "duplicate".  We get
 // all sorts of things from different providers/transcoders, so in the end, we
-// just settled on PTS and segmentation type
+// just settled on PTS, event id, and segmentation type
```

Comment:
I added this because in the out-out-in case, it's really out-out-in-in where the inner ones are linked by event id.  Overall it seems like a good id to use event id because so many signals are linked by them.  If we process the same signal twice and it has different event ids, I don't think it would cause any issues.  In the case of multiple outs of the same type for the same pts with different event ids (which are supposed to be the same) the previous out will be closed by the next repitition.